amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -560,9 +560,11 @@ type ColonyAction @model {
   blockNumber: Int!
   isMotion: Boolean
   motionData: MotionData
+  showInActionsList: Boolean! # True for (forced) actions. True for motions if staked above 10%
   createdAt: AWSDateTime!
+  initiatorAddress: ID!
```

Comment:
I am relying on this being present to determine whether to filter out an action from the actions list. If the initiatorAddress is a voting reputation extension, then it shouldn't appear. 

Therefore, making compulsory. Requires no downstream changes since it was already present on all db mutations, as far as I can see.